app/views/carts/show.html.erb

```diff
@@ -5,6 +5,7 @@
   <%= item.title %>
   <%= item.description %>
   <%= item.price %>
+  <%= link_to "Remove", cart_path(item), method: :delete %>
```

Comment:
Why are we passing item to the cart path? Doesn't the cart path just display the cart? Why does it need an argument?

Reply:
It needs the item in this remove link so that it knows which item to take in the params to delete. We don't want to delete all items from the cart, we just want to delete a certain item and have the cart quantity and total price to adjust correctly.

Reply:
Awesome. Makes sense. Looks good to me.